tests/e2e/config/bootstrap.js

```diff
@@ -122,6 +122,38 @@ function observeConsoleLogging() {
 	} );
 }
 
+/**
+ * Runs Axe tests when the block editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForBlockEditor() {
+	if ( ! await page.$( '.block-editor' ) ) {
+		return;
+	}
+
+	await expect( page ).toPassAxeTests( {
+		// Temporary disabled rules to enable initial integration.
+		// See: https://github.com/WordPress/gutenberg/pull/15018.
+		disabledRules: [
+			'aria-allowed-role',
```

Comment:
These rules are disabled because of the errors in gutenberg. I would love for this ignored rules to be removed.